drivers/storage/portworx/preflight.go

```diff
@@ -201,8 +201,8 @@ func (u *preFlightPortworx) RunPreFlight() error {
 	checkArgs(u.podSpec.Containers[0].Args)
 
 	if !u.hardFail {
-		// If Dmthin param does not exist add it
-		preflightDS.Spec.Template.Spec.Containers[0].Args = append([]string{"-T", "dmthin"},
+		// If PX-StoreV2 param does not exist add it
+		preflightDS.Spec.Template.Spec.Containers[0].Args = append([]string{"-T", "PX-storeV2"},
```

Comment:
please check the case-sensitivity

Reply:
@vumesh-px I am sorry I missed this comment 🙏.   Did not realize it until today.    Should be fixed now.